unlock-app/src/__tests__/components/content/LogContent.test.tsx

```diff
@@ -67,4 +82,33 @@ describe('Transaction Log', () => {
       )
     })
   })
+
+  jest.mock('next/router', () => {})
```

Comment:
can you move the jest.mock call to the top of the file just below imports? This is consistent with other tests